sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -1336,16 +1336,6 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
 ! GFS output dust in nemsio (GOCART)
         dustcb=0.0
         dustallcb=0.0
-        do n=1,nbin_du
```

Comment:
@zhanglikate I would like to confirm that initializing variables dust, salt, soot, waso, suso, pp25 and pp10 are removed as ALLOCATE_ALL is functioning that.

Reply:
Yes. They are all defined and initialized ALLOCATE_ALL. Thanks.